packages/hash/shared/src/types.ts

```diff
@@ -1,393 +1,14 @@
-import { BaseUri, VersionedUri } from "@blockprotocol/type-system";
-import slugify from "slugify";
-import { systemUserShortname } from "@hashintel/hash-shared/environment";
-import { frontendUrl } from "./environment";
+// The types exported here are expected to be branded.
+// Perhaps some of these should be put into the subgraph stdlib instead?
 
-type SchemaKind = "data-type" | "property-type" | "entity-type";
+export type ownedById = string;
```

Comment:
Do we want to add
```ts
export type uuid = string
```

And change these to use that?

Reply:
Sure we can do that for now